src/DynamoCoreWpf/Controls/DynamoTextBox.cs

```diff
@@ -153,6 +153,11 @@ public bool Pending
         new public string Text
         {
             get { return base.Text; }
+            set
+            {
+                base.Text = value;
+                UpdateDataSource(true);
+            }
```

Comment:
The setter was left out (and new `Text` property created to hide the base property) intentionally, because that messes with the sequence in which undo recording is done. Is there a reason we add this now? Undo around the use of `DynamoTextBox` must be validated after this.